Lib/glyphsLib/builder/glyph.py

```diff
@@ -256,6 +257,41 @@ def to_glyphs_glyph(self, ufo_glyph, ufo_layer, master):  # noqa: C901
     self.to_glyphs_paths(ufo_glyph, layer)
     self.to_glyphs_components(ufo_glyph, layer)
     self.to_glyphs_glyph_anchors(ufo_glyph, layer)
+    self.to_glyphs_glyph_height_and_vertical_origin(ufo_glyph, master, layer)
+
+
+def to_ufo_glyph_height_and_vertical_origin(self, ufo_glyph, layer):
+    # implentation based on:
+    # https://github.com/googlefonts/glyphsLib/issues/557#issuecomment-667074856
+    if not self.is_vertical:
```

Comment:
I would rather place this check for `is_vertical` immediately before the method is called, and assume (assert) it is within the body of the current method

Reply:
the short form of the vhea values is without the Typo: vheaVertAscender, vheaVertDescender, vheaVertLineGap

Reply:
Thanks. Sono guess we should support both

Reply:
Thanks Georg.

Cosimo, if I move `if self.is_vertical` to the parent function, I get the following flake8 error:

`Lib/glyphsLib/builder/glyph.py:29:1: C901 'to_ufo_glyph' is too complex (20)`

I know the `to_ufo_glyph` function is a smorgasbord of conditionals but I feel this test is perhaps a bit too limiting?

Reply:
you can probably mute flake8 for that function only. Try adding `# noqa: C901` to disable that check for this particular method.